src/RobloxRenderer.spec.lua

```diff
@@ -541,6 +542,25 @@ return function()
 			expect(spyRef.callCount).to.equal(2)
 			spyRef:assertCalledWith(nil)
 		end)
+
+		itSKIP("should not process events when unmounting", function()
```

Comment:
Should this still be skipped? If it's not currently passing, should it get to passing before being merged?

Reply:
I am not sure what I should do with this, because the test passes when it's running in studio, but not using Lemur, because it does not simulate the ChildRemoved event. So I don't know if I should implement the feature in Lemur or if there is another way to test this!

Reply:
@ZoteTheMighty how should we handle this?